plugin/src/main/resources/io/jenkins/plugins/coverage/metrics/restapi/ModifiedLinesCoverageApi/_api.jelly

```diff
@@ -0,0 +1,17 @@
+<?jelly escape-by-default='true'?>
```

Comment:
This file needs to be moved to `[...]/coverage/metrics/restapi/CoverageApi/_api.jelly`. It should be shown when the user clicks on the API link on the `coverage` view.

Reply:
Ah, I see. Thanks for the hint.